staffing/views.py

```diff
@@ -581,7 +591,10 @@ def prod_report(request, year=None, month=None):
             status = all_status["ko"]
         else:
             status = all_status["ok"]
-        totalData.append([status, "", [formats.number_format(turnover), formats.number_format(forecast)]])
+        if month.year == date.today().year and month.month == date.today().month:
```

Comment:
could you please factorize the "current month" detection ? (is_current_month true/false) ?